### ISSUE TITLE
Three drugs from NCI60 failed to map to the drugs_with_ids.csv file

### ISSUE THREAD

**Issue by ChristopherEeles** [bhklab/PharmacoDI_snakemake_pipeline#8]
These drugs ended up with NULL compound_uid values in the database. Since compound_uid is being used to make the compound page URLs, we cannot have NULLs in there.

The drugs are:
illudin m linked to """"fleximer""""
""""Ether-20""""
N/A.

They all come from NCI60. The first two fail to map due to differing numbers of quotations in the drug names. The third one, N/A. - which OMG who let this be a valid drug synonynm - seems to have been renamed in the drugs_with_ids.csv file.

SOPs need to be put in place to ensure names always match between drugs_with_ids.csv and the current PSet versions.

It also seems like we need some SOPs around quotations in our unique.drugids. I would propose that we remove all quotations inside drug names, so that quotations in a .csv file
indicate a string.

**Comment by sisiranair** [comment]
I agree with removing quotations (we discussed this earlier, yet to implement)
Nikta might have to update the pset names as per the recent changes to N/A. and if we decide to remove quotations.
